language/README.md

```diff
@@ -2,22 +2,56 @@
 
 # Google Cloud Natural Language API Ruby Samples
 
-[Cloud Natural Language API][language_docs] provides natural language
+The [Google Cloud Natural Language API][language_docs] provides natural language
 understanding technologies to developers, including sentiment analysis, entity
 recognition, and syntax analysis. This API is part of the larger Cloud Machine
```

Comment:
I think the "Cloud Machine Learning API" language is outdated. I would just delete this line.